build/scripts/migrate.sh

```diff
@@ -32,6 +32,7 @@ fi
 if [[ "X${FORCE_MIGRATE}" == "Xtrue" ]];
 then
     echo "Running the Solidity contracts migrations for local geth"
+    sleep 30 # allow geth block gas limit to increase to more than 7000000
```

Comment:
Is this the fix to wait until at least one block mined so the contracts can be created? I saw some code with a loop too. one or the other might be redundant?

Reply:
the other one is for local, above is for travis. For local there may be a docker geth running already so the retry works instead of just waiting unnecessarily. Ideally we should query geth for the block gas limit, but I found that creates other issues, above seems simple enought